test/test.cc

```diff
@@ -438,6 +439,14 @@ void build_live_traffic_data(const boost::property_tree::ptree& config,
   std::string tile_dir = config.get<std::string>("mjolnir.tile_dir");
   std::string traffic_extract = config.get<std::string>("mjolnir.traffic_extract");
 
+  // Create parent path before creating live traffic extract
+  filesystem::path parent_dir = filesystem::path(traffic_extract).parent_path();
+  if (!filesystem::exists(parent_dir)) {
```

Comment:
@danpat I've changed it to throw exception if the traffic extract's parent directory(s) does not exist. That will force the user to specify the same dir as that for tiles, instead of creating new directories everywhere.